generators/slack.py

```diff
@@ -11,29 +11,44 @@ class SlackMessageGenerator(generators.base.BaseMessageGenerator):
     # We will use this constant to split the message blocks into smaller ones
     SLACK_MAX_MESSAGE_BLOCK_SIZE = 3000
 
+    NO_LEVELS_IN_SCORECARD_MESSAGE = lambda title: [
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": "There are no levels in {} scorecard :disappointed:".format(title)
+            }
+        }
+    ]
```

Comment:
is this something you've experienced? as this shouldn't happen that there are no levels in scorecard